public/index.html

```diff
@@ -124,7 +124,6 @@ <h1 class="hb qi ui xi _i ok sl hm no kp">
                   ergonomique et fiable.
                 </p>
 
-                <div class="bc je le">
```

Comment:
You probably broke the tree here, that div isn't closed.
CI should complain about it too